gravitee-apim-portal-webui-next/src/app/api-details/api-tab-documentation/api-tab-documentation.component.ts

```diff
@@ -68,7 +69,12 @@ export class ApiTabDocumentationComponent implements OnInit {
   private getSelectedPage$(pageId: string): Observable<SelectedPageData> {
     return this.pageService.getByApiIdAndId(this.apiId, pageId, true).pipe(
       map(result => ({ result })),
-      catchError(error => of({ error })),
+      catchError((error: HttpErrorResponse) => {
```

Comment:
Later on we can consider adding `global error handler / interceptor` to our app, to basically catch all e.g 404 errors without adding extra code

Reply:
Totally!
Just a note for this case: the page id is a query param, so if a page does not exist for that id, then nothing happens. This is a way to force it to reroute to the 404 page.